.env

```diff
@@ -26,12 +26,25 @@ EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
 # option, which could potential exclude all nodes from load balancer at the same time.
 MAX_CONCURRENT_UPDATE=1
 
-# Update time window setting
+# Update time window setting - DEPRECATED
+# Note!! that these settings are deprecated and you should use the Scheduler settings below.
 # Brupop will operate node updates within update time window.
 # when you set up time window start and stop time, you should use UTC (24-hour time notation).
 UPDATE_WINDOW_START=0:0:0
 UPDATE_WINDOW_STOP=0:0:0
 
+# scheduler settings
+# Brupop will operate node updates on scheduled maintenance windows by using cron expressions.
+# When you set up the scheduler, you should follow cron expression rules.
```

Comment:
Just for brevity, can we also add the following to our documentation?
```
# ┌───────────── seconds (0 - 59)
# | ┌───────────── minute (0 - 59)
# | │ ┌───────────── hour (0 - 23)
# | │ │ ┌───────────── day of the month (1 - 31)
# | │ │ │ ┌───────────── month (Jan, Feb, Mar, Apr, Jun, Jul, Aug, Sep, Oct, Nov, Dec)
# | │ │ │ │ ┌───────────── day of the week (Mon, Tue, Wed, Thu, Fri, Sat, Sun)
# | │ │ │ │ │ ┌───────────── year (formatted as YYYY)
# | │ │ │ │ │ |
# | │ │ │ │ │ |
# * * * * * * *
```

Some systems and tools implement their own wonky cron expression handlers. For example, I had assumed I `?` was standard to define a day of the month but it's not. Let's get as close to documenting what [the cron library expects](https://github.com/zslayton/cron)

Reply:
maybe better to add this to README?